src/components/Staff/MinStaffForm.tsx

```diff
@@ -0,0 +1,76 @@
+import React, {useState} from "react";
+import {Alert, Box, Button, OutlinedInput} from "@mui/material";
+import CheckCircleIcon from '@mui/icons-material/CheckCircle';
+
+export interface IMinStaffForm {
+  port: string
+  terminal: string
+  minStaffNumber: number | null
+  handleSubmit: (minStaff: number | null) => boolean
+  continueCallback: () => void
+}
+
+export const MinStaffForm = ({port, terminal, minStaffNumber, handleSubmit, continueCallback}: IMinStaffForm) => {
+  const [staffNumber, setStaffNumber] = useState<number | null>(minStaffNumber);
+  const [error, setError] = useState(false);
+  const [updated, setUpdated] = useState(false);
```

Comment:
Maybe this could be called `submitted` as it's set once the form has been submitted